src/cockpit/channel.py

```diff
@@ -248,7 +248,6 @@ async def create_transport(self, loop: asyncio.AbstractEventLoop, options: Dict[
         raise NotImplementedError
 
     def do_open(self, options):
-        self.freeze_endpoint()
```

Comment:
This landed in the wrong commit.  Let's let the bots run before moving it, though.